baseband/mark4/base.py

```diff
@@ -245,40 +244,40 @@ def write_frame(self, data, header=None, **kwargs):
 class Mark4StreamReader(VLBIStreamReaderBase, Mark4FileReader):
     """VLBI Mark 4 format reader.
 
-    This wrapper is allows one to access a Mark 4 file as a continues series
-    of samples.  Note that possible gaps in the data stream are not filled in,
-    though parts of the data stream replaced by header values are filled in.
+    Allows access to a Mark 4 file as a continuous series of samples.  Note
+    that possible gaps in the data stream are not filled in, though parts of
+    the data stream replaced by header values are filled in.
 
     Parameters
     ----------
-    fh_raw : `~baseband.mark4.Mark4FileReader`
-        File handle to the raw Mark 4 data stream.
+    fh_raw : filehandle
+        filehandle of the raw Mark 4 stream.
+    sample_rate : `~astropy.units.Quantity`, optional
+        Number of complete samples per second (i.e. the rate at which each
+        channel is sampled).  If not given, will be inferred from scanning two
+        frames of the file.
     ntrack : int, or None
-        Number of tracks used to store the data.  If ``None``, will attempt to
-        automatically detect it by scanning the file.
+        Number of Mark 4 bitstreams.  If `None`, will attempt to automatically
```

Comment:
would write `optional` above, and "If not given" here

Reply:
For consistency with the above, shouldn't we remove all `optional` from header keywords?